core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -185,18 +216,60 @@ public PlayerInformation getPlayerInformation() {
     public void attach(PlayerView playerView) {
         videoContainer = VideoContainer.with(playerView.getContainerView());
         playerView.simplePlayerView().setPlayer(exoPlayer);
-        // TODO : Set SubtitleView on the facade.
     }
 
     @Override
-    public void selectAudioTrack(int audioTrackIndex) {
-        //TODO: Select the audio track on the facade.
+    public void selectAudioTrack(PlayerAudioTrack audioTrack) {
+        MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
+        TrackGroupArray trackGroups = trackInfo.getTrackGroups(TRACK_TYPE_AUDIO);
+        FixedTrackSelection.Factory factory = new FixedTrackSelection.Factory();
+
+        MappingTrackSelector.SelectionOverride selectionOverride = new MappingTrackSelector.SelectionOverride(
+                factory,
+                audioTrack.groupIndex(),
+                audioTrack.trackIndex()
+        );
+        trackSelector.setSelectionOverride(TRACK_TYPE_AUDIO, trackGroups, selectionOverride);
     }
 
     @Override
     public List<PlayerAudioTrack> getAudioTracks() {
-        //TODO : Get a list of audio tracks from the facade.
-        return Collections.emptyList();
+        MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
```

Comment:
Same with all this code, if we could extract all this into an external class it will make our lives easier :)